compiler/compile-config/compile-config-2.02-learning_inference

```diff
@@ -197,10 +197,11 @@ if .deepdive_.execution.processes | has("process/grounding/combine_factorgraph")
                 cd \"$DEEPDIVE_RESULTS_DIR\"/$pid
                 cat inference_result.out.text |
                 # restoring shard ID to the vids
-                jq -R -r --argjson SHARD_BASE $(($pid << 48)) '
-                    split(\" \") |
-                    [ (.[0] | tonumber + $SHARD_BASE | tostring)
-                    , .[1:][] ] | join(\"\\t\")' |
+                python3 -c "
```

Comment:
I couldn't resist doing a [benchmark notebook](https://gist.github.com/netj/20e00e1719684ce08f38edff519f575a)
and the winner is.... Perl!

jq was actually the slowest, ~7x slower than Perl and 3x than AWK.  Python is ~1.7x, AWK is ~2.2x.
jq and AWK uses little memory ~10MB, ~1/3 of Python (35-45MB), but so does Perl (14MB).
So Perl is a clear winner.